docs/requirements.rst

```diff
@@ -135,7 +131,5 @@ DeepChem has a number of "soft" requirements.
 .. _`RDKit`: http://www.rdkit.org/docs/Install.html
 .. _`simdna`: https://github.com/kundajelab/simdna
 .. _`Tensorflow Probability`: https://www.tensorflow.org/probability
-.. _`XGBoost`: https://xgboost.readthedocs.io/en/latest/
```

Comment:
Should we add lightgbm/xgboost as soft requirements here? Since you'd need one or the other to use the model